### ISSUE TITLE
cryptsetup: Permision denied

### ISSUE THREAD

**Issue by NotDarkn** [FWNavy/CryptoSmite#9]
## Built Using?
EndeavourOS with KDE Plasma 6.0.1

## Describe your Issue?
Upon using the latest current commit of the reenrollment branch (6d15131), it reports "permission denied!" when ]trying to use cryptsetup.
```
Setting up encstateful
/usr/local/bin/cryptosmite.sh: line 112: cryptsetup: Permission denied
cryptsetup open --type plain --cipher aes-cbc-essiv:sha256 --key-size 256 --key-file "$ENCSTATEFUL_KEY" "$STATEFUL_MNT"/encrypted.block encstateful failed with exit code 127.
umount: no mount point specified.
/usr/local/bin/cryptosmtie.sh: line 60: cryptsetup: Permission denied
```
## Conclusion
The `cryptosmite.sh` needs to run `chmod +x` on both `/usr/local/bin/cryptsetup_aarch64` and `/usr/local/bin/cryptsetup_x86_64`, that's about it. 

When it does, the `cryptosmite.sh` file works as it should and unenrolls the Chromebook.

**Comment by NotDarkn** [comment]
made pr #8

**Comment by MunyDev** [comment]
Fixed #9 by merge to reenrollment branch
